### DISCUSSION TITLE
Replit - No Persistence?

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#1868] [Q&A]
I have Flowise installed as a package in Replit. If I run it, add Chatflows and save them, and then log out and log back in to Replit and restart Flowise then those Chatflows are missing. I guess the Chatflow data is stored in a file or folder which Replit does not persist across VM restarts. Has anyone encountered else encountered this?

**Comment by Jaredude** [comment]
Without having persistent storage or  pointing Flowise to a DB, this will be the expected behavior with Replit or any other cloud provider.

**Comment by mikehudson2** [comment]
For anyone else who struggled with this, my experience was:

1. The Render instructions on https://docs.flowiseai.com/ worked perfectly and as described you need to use a "Disk" to persist Flowise data.

2. The Replit instruction didn't work unless you also do https://ask.replit.com/t/flowise-install/69905/5. Within the IDE mode Flowise data persists when Flowise is stopped and started BUT is wiped when you log out and back in to Replit.

The Render and Railway instructions are explicit of the ephemeral Flowise data and on how to persist it. I couldn't see any guidance on how to do something similar in the Replit instructions.
